### PR TITLE
[openxr-loader] Update to 1.0.20

### PR DESCRIPTION
**Pull request by RT2Code**
Update the openxr-loader port from 1.0.18 to 1.0.20.

Changelogs:
https://github.com/KhronosGroup/OpenXR-SDK/releases/tag/release-1.0.19
https://github.com/KhronosGroup/OpenXR-SDK/releases/tag/release-1.0.20

### CODE REVIEW

File: ports/openxr-loader/portfile.cmake [hunk 1]
```diff
@@ -44,9 +44,8 @@ endif()
 
 vcpkg_find_acquire_program(PYTHON3)
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (JonLiu1993) [inline ports/openxr-loader/portfile.cmake:48]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

File: ports/openxr-loader/portfile.cmake [hunk 2]
```diff
@@ -70,9 +69,9 @@ foreach(HEADER ${HEADER_LIST})
...
 
 if(VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/OpenXR)
+    vcpkg_cmake_config_fixup(CONFIG_PATH cmake)
 else(VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/openxr TARGET_PATH share/OpenXR)
+    vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/openxr)
 endif()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
```
Comment (JonLiu1993) [inline ports/openxr-loader/portfile.cmake:77]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```

Comment (JonLiu1993) [reply ports/openxr-loader/portfile.cmake:77]: @RT222 ,Thanks for your pr, we should add double quotes to the absolute path in the portfile.cmake like this

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@RT222 ,Have you test the features locally?

**Comment by RT222** [comment]
@JonLiu1993 Yes, I tested it on the following platforms :

- x86-windows
- x64-windows
- x64-windows-static

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**